domains/zitadokm.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "imadkm2006",
+        "email": "imadzm078@gmail.com"
+    },
+    "record": {
+        "TXT": "google-site-verification=g3p8xinebwx8oji9sagwy8nvydzhovmxeru6l6dvdkc"
```

Comment:
Google site verification through TXT record is not supported, use the HTML method
also you need some contents on your site (unless it's for mails)